### PR TITLE
EDU-3315: Update Pricing doc clarification

### PR DESCRIPTION
**Pull request by drewhoskins-temporal**
## What does this PR do?
* Clarifies Update pricing in advance of GA.
* Eliminates outdated messaging of "an update is a query plus a signal"

### CODE REVIEW

File: docs/evaluate/temporal-cloud/pricing.mdx [hunk 1]
```diff
@@ -77,7 +77,7 @@ They track billable operations within the Temporal Cloud Service, such as starti
 - **Side Effect recorded**.
   For a mutable [Side Effect](/workflows#side-effect), an Action occurs only when the value changes.
   (Be aware that some SDKs don't support Side Effects.)
-- [**NEW**] **Workflow Update**. [Workflow Updates](/encyclopedia/workflow-message-passing#sending-updates) is a primitive that combines a Signal and Query together for a single Action.
+- **Workflow Update received**. An [Update](/encyclopedia/workflow-message-passing#sending-updates) is received when you handle it or when your [Update Validator](/encyclopedia/workflow-message-passing#update-validators) rejects it.
```
Comment (alexshtin) [inline docs/evaluate/temporal-cloud/pricing.mdx:80]: `processed`? I use processed for "accepted/rejected" on server side.

**Review by alexshtin — APPROVED** [review]

File: docs/evaluate/temporal-cloud/pricing.mdx (hunk 1 above)
Comment (fairlydurable) [reply docs/evaluate/temporal-cloud/pricing.mdx:80]: @drewhoskins-temporal Is it fair to say that "An Action occurs when an [Update](....) is received, when you handle it, or when your [Update Validator](....) rejects an update"?

Comment (fairlydurable) [reply docs/evaluate/temporal-cloud/pricing.mdx:80]: (And if so, does that mean that you get 2 actions -- receive and handle, or 2 actions -- receive and reject?)

Comment (drewhoskins-temporal) [reply docs/evaluate/temporal-cloud/pricing.mdx:80]: @alexshtin 
Signal says "sent" 
Queries says "received"
I don't want Updates to use a _third_ word like "processed." 

I would prefer to use the same word for all of them if it's technically accurate.

Comment (fairlydurable) [inline docs/evaluate/temporal-cloud/pricing.mdx:80]:
```suggestion
- **Workflow Update received**. An Action occurs when an [Update](/encyclopedia/workflow-message-passing#sending-updates) is received by a Worker and either processed by the Update handler, or rejected by an [Update Validator](/encyclopedia/workflow-message-passing#update-validators).
```

**Review by dandavison — APPROVED** [review]

**Review by fairlydurable — APPROVED** [review]
- No issues found.
- Links checked, all go to the right places.

Unblocking.

**Pull request merged**